integration-tests/http/__tests__/product/store/product.spec.ts

```diff
@@ -401,5 +469,640 @@ medusaIntegrationTestRunner({
         expect(response.status).toEqual(200)
       })
     })
+
+    describe("GET /store/products", () => {
```

Comment:
Were these tests aimed at v2 only since the beginning? I don't see any `BREAKING` comments so maybe that's why, I am sure we do have breaking changes though 😄

Reply:
yeah, this is coming in from modules, so v2 specific. We surely do, but those will come in when the store products tests are ported over from api